eppo_client/client.py

```diff
@@ -83,17 +169,28 @@ def get_assignment(
         )
         assigned_variation = next(
             (
-                variation.value
+                variation
                 for variation in allocation.variations
                 if is_in_shard_range(shard, variation.shard_range)
             ),
             None,
         )
+
+        assigned_variation_value_to_log = None
+        if assigned_variation is not None:
+            assigned_variation_value_to_log = assigned_variation.value
```

Comment:
I debated doing this as well (using `value` for logging). Maybe I should have. Keeps things cleaner.